src/features/tokens/price-service.test.js

```
@@ -0,0 +1,32 @@
+import {getCoinCapToken, emptyResponse} from './price-service';
+
+describe('Price service', () => {
+  describe('getCoinCapToken', () => {
+    const successResponse = {
+      priceUsd: 200,
+    };
+
+    it('expect to return success response', async () => {
+      global.fetch = () =>
+        Promise.resolve({
+          status: 200,
+          json: () => Promise.resolve({data: successResponse}),
+        });
+
+      const result = await getCoinCapToken('DOCK');
+
+      expect(result).toBe(successResponse);
+    });
+
+    it('expect to handle error', async () => {
+      global.fetch = () =>
+        Promise.resolve({
+          status: 400,
+        });
+
+      const result = await getCoinCapToken('DOCK');
+
+      expect(result).toBe(emptyResponse);
```

should we return more details or throw an exception or something instead of an empty response? An empty response can be ambiguous between "no data found" and "an error occurred". If someone is consuming this SDK and needs to know to show an error to the user it might be easier with an error or a response that includes error details.